voluncheer/profiles/models.py

```diff
@@ -125,6 +127,37 @@ def __str__(self):
         return self.name
 
 
+class BadgeType(models.IntegerChoices):
+    """Container for all possible badge types."""
+
+    VOLUNTEER_LEVEL = 0, "VOLUNTEER_LEVEL"
```

Comment:
So, the original name for this "HOURS_REQUIRED" as in a badge that is awarded if you volunteer a certain amount of required hours, but I thought _that_  was confusing. The idea behind volunteer level is is that it an overarching level for your account...so if you volunteer X hours you are bronze level, if you volunteer Y hours you are silver level, etc..

Happy to change it to whatever you think is more readable.

Reply:
Regarding when volunteers are awarded this type of badge, right now it checks whenever if you refresh your profile page if you have volunteered enough hours. In the future, this would happen just on login.

You only get a badge if your `hours_volunteered `exceeds the `hours_required `for that badge.

Reply:
Right, but Volunteer level is associated with only time-spent-volunteering badge, not other badges. I think instead of volunteer_level, it should be volunteer_hour_badge. Others could be dedicated_service_badge for example.

Reply:
Will do.